### DISCUSSION TITLE
MCP servers for all public data with OpenDataMCP

### DISCUSSION THREAD

**Discussion by grll** [modelcontextprotocol/.github#57] [Show and tell]
### Pre-submission Checklist

- [X] This post follows the MCP community guidelines

### What would you like to share?

With [Open Data MCP](https://github.com/OpenDataMCP/OpenDataMCP) we have the ambition to bring as many publicly available datasets to Claude and other LLMs as possible via the MCP protocol.

Today we offer:

* a CLI tool that allows you to one-command setup any of our mcp servers on the Claude Desktop app.
* a initial template idea that allows you to quickly get going with developing your mcp server.
* Data through "MCP Tools" for now of the swiss railway service (3 datasets). And we will also soon add Swiss weather data from local weather stations probably as "MCP Resources".

As we are Swiss based, we started by focusing on the data we know well: swiss public data but we don't want to restrict anyone to contribute with their own public data from anywhere.

We are just getting started and a lot remains to be figured out. We are actively looking for developers to help us build this connected future by bringing support to more public datasets and/or developing new features with us.

It's very early, we see a lot of challenges to come but we are very excited! let's build together!

### Relevant Links

- https://github.com/OpenDataMCP/OpenDataMCP

**Comment by viscoussnake** [comment]
Brilliant idea
